src/test/file-utils.test.js

```diff
@@ -3,53 +3,67 @@ const fileUtils = require('../main/lib/file-utils');
 
 describe('file-utils', () => {
 
-  describe('listFilesInDirectory', () => {
+  describe('getJsonFilePaths', () => {
     it('lists all files in the directory if no filters are provided', () => {
-      const filesInDirectory = fileUtils.listFilesInDirectory(
-        './src/test/fixtures/listFiles'
-      );
+      const filesInDirectory = fileUtils.getJsonFilePaths('./src/test/fixtures/listFiles');
 
       assert.equal(filesInDirectory.length, 2);
-      assert.equal(filesInDirectory[0].name, 'NotExcluded.json');
-      assert.equal(filesInDirectory[1].name, 'UserProfile.json');
+      assert.equal(filesInDirectory[0], 'NotExcluded.json');
+      assert.equal(filesInDirectory[1], 'UserProfile.json');
     });
 
-    it('lists all files in the directory if filters are not matching any of the file', () => {
-      const filesInDirectory = fileUtils.listFilesInDirectory(
-        './src/test/fixtures/listFiles', ['UserProfile']
-      );
+    it(
```

Comment:
Please inline it as it was before. For me there is no point having `it` on a separate line.